app/routes/admin/content/system-images.js

```diff
@@ -5,5 +5,39 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('System Images');
+  },
+
+  model() {
```

Comment:
@niranjan94 Do you want me to move that inner model loop outside and the subTopics loop inside? If we do that, what happens is for ex if we do not get data for any subTopic from API, then the subTopic won't be visible, but we want all subTopics from the dicrtionary to be appear on the page. Please guide.